apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -478,7 +482,8 @@ var removeWatermark = function (focusEvent) {
  * @returns {boolean} true iff the given event represents a clean enter
  */
 var isUnmodifiedEnterPress = function (jqueryEvent) {
-  return (jqueryEvent.keyCode == 13 && !(jqueryEvent.ctrlKey || jqueryEvent.shiftKey));
+  return (jqueryEvent.keyCode == KeyCodes.ENTER &&
```

Comment:
I don't think so!  Didn't even notice when I dropped the constant in; nice catch.  I should update my file-specific jshint settings to warn about these.